swift/build_ffi.sh

```diff
@@ -15,7 +15,7 @@ export CARGO_PROFILE_RELEASE_DEBUG=1 # enable line tables
 
 usage() {
   cat >&2 <<END
-Usage: $(basename "$0") [-d|-r] [-v] [--generate-ffi|--verify-ffi]
+Usage: $(basename "$0") [-d|-r] [-v] [--generate-ffi|--verify-ffi|--use-xargo]
```

Comment:
Is this mutually exclusive with the others? …I guess it probably is, since we're not set up to generate headers with xargo.

Reply:
Yeah, cbindgen doesn't work correctly with the targets that require being built from source (using xargo)